### PR TITLE
add  MY social account to Link tree 

### PR DESCRIPTION
**Pull request by AjRadwan**

### CODE REVIEW

File: .github/public/data/AjRadwan.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://github.com/AjRadwan",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Anowar_J_Radwan",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
[the remark this answers is not in the document]
Comment (AjRadwan) [reply .github/public/data/AjRadwan.json:18]: what is the problem here?

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the `package-lock.json` and `package.json` files. Let us know if you need any help with that! 😄

**Comment by AjRadwan** [comment]
> Please revert the changes in the `package-lock.json` and `package.json` files. Let us know if you need any help with that! 😄

I didn't even touch that file thought. well if needed I am gonna revert it  (:

**Comment by AjRadwan** [comment]
> Please revert the changes in the `package-lock.json` and `package.json` files. Let us know if you need any help with that! 😄
okay. I didn't change anything in these two files. so how should I revert it?
 should I go to specific files and revert them or should I revert all? and commit again.

**Comment by AjRadwan** [comment]
I always use git add.  I didn't realize it can cause a problem like that. thank you for the solution

**Review by emmalearnscode — APPROVED** [review]
This looks good to me, thanks for removing the extra files 👍🏻

**Review by KendallDoesCoding — APPROVED** [review]
LGTM, Only thing I would suggest is make the I caps in LinkedIn, but as I said, there's no harm with that being as it is too :)

**Comment by AjRadwan** [comment]
> LGTM, Only thing I would suggest is make the I caps in LinkedIn, but as I said, there's no harm with that being as it is too :)

yap tomorrow this problem happened because I change the cap hahaha, I am gonna change it again :)

**Comment by KendallDoesCoding** [comment]
> > LGTM, Only thing I would suggest is make the I caps in LinkedIn, but as I said, there's no harm with that being as it is too :)
> 
> yap tomorrow this problem happened because I change the cap hahaha, I am gonna change it again :)

Ok thanks.

Also, it's not "tommrow" it's "yesterday" :)

**Comment by KendallDoesCoding** [comment]
> > LGTM, Only thing I would suggest is make the I caps in LinkedIn, but as I said, there's no harm with that being as it is too :)
> 
> yap tomorrow this problem happened because I change the cap hahaha, I am gonna change it again :)

Ok thanks.

Also, it's not "tommrow" it's "yesterday" :)

**Comment by KendallDoesCoding** [comment]
> > LGTM, Only thing I would suggest is make the I caps in LinkedIn, but as I said, there's no harm with that being as it is too :)
> 
> yap tomorrow this problem happened because I change the cap hahaha, I am gonna change it again :)

Ok thanks.

Also, it's not "tommrow" it's "yesterday" :)

**Comment by AjRadwan** [comment]
> Also, it's not "tomorrow" it's "yesterday" :)

I can't believe I wrote tomorrow. (:

**Comment by KendallDoesCoding** [comment]
> > Also, it's not "tomorrow" it's "yesterday" :)
> 
> I can't believe I wrote tomorrow. (:

no problem, was just letting you know, i understood what you meant to say : D

**Comment by KendallDoesCoding** [comment]
Any update on this PR?

File: .github/public/data/AjRadwan.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/AnowarJRadwan",
+      "icon": "facebook"
+    }
+  ]
+}
+
+
```
Comment (Panquesito7) [inline .github/public/data/AjRadwan.json:35]: Should we fix the endline or let the `github-actions` bot fix it?

Comment (KendallDoesCoding) [reply .github/public/data/AjRadwan.json:35]: anything works!

**Review by Panquesito7 — APPROVED** [review]

File: .github/public/data/AjRadwan.json (hunk 2 above)
Comment (krshdev) [reply .github/public/data/AjRadwan.json:35]: I think github-actions should do :)

**Pull request merged**